micro-couchbase/src/main/java/com/aol/micro/server/couchbase/ConfigureCouchbase.java

```diff
@@ -45,15 +45,26 @@
     @Value("${couchbaseClientOperationTimeout:120000}")
     private long opTimeout;
 
+    @Value("${distributed.cache.default.expiration:691200}")
+    private int expiresAfterSeconds = 691200;
```

Comment:
Don't need to assign this value will be overwritten  by Spring annotation same for line 52,55